<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="detalle_cto_certificadosV2-20241022160336" tests="1" file=".py" time="104.411" timestamp="2024-10-22T16:05:20" failures="0" errors="1" skipped="0">
	<testcase classname="detalle_cto_certificadosV2" name="test_detail_certificates" time="104.411" timestamp="2024-10-22T16:05:20" file="CtaCtoDtalCertificado2.py" line="22">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=130.0.6723.58)
Stacktrace:
	GetHandleVerifier [0x00007FF72805E1F5+2972373]
	(No symbol) [0x00007FF727CF55F0]
	(No symbol) [0x00007FF727B957FA]
	(No symbol) [0x00007FF727B6FA85]
	(No symbol) [0x00007FF727C12B17]
	(No symbol) [0x00007FF727C2B1D1]
	(No symbol) [0x00007FF727C0B823]
	(No symbol) [0x00007FF727BD75E8]
	(No symbol) [0x00007FF727BD8751]
	GetHandleVerifier [0x00007FF7280847BD+3129501]
	GetHandleVerifier [0x00007FF7280D4D00+3458528]
	GetHandleVerifier [0x00007FF7280CB05D+3418429]
	GetHandleVerifier [0x00007FF727E5687B+844123]
	(No symbol) [0x00007FF727D00AFF]
	(No symbol) [0x00007FF727CFC6D4]
	(No symbol) [0x00007FF727CFC86D]
	(No symbol) [0x00007FF727CEBD79]
	BaseThreadInitThunk [0x00007FF92EF0257D+29]
	RtlUserThreadStart [0x00007FF93006AF08+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\CtaCtoDtalCertificado2.py", line 95, in test_detail_certificates
    find_elements(self.driver, apply_filter )
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\Elements.py", line 19, in find_elements
    select_nex_button = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 363, in _predicate
    target = driver.find_element(*target)  # grab element at locator
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 748, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=130.0.6723.58)
Stacktrace:
	GetHandleVerifier [0x00007FF72805E1F5+2972373]
	(No symbol) [0x00007FF727CF55F0]
	(No symbol) [0x00007FF727B957FA]
	(No symbol) [0x00007FF727B6FA85]
	(No symbol) [0x00007FF727C12B17]
	(No symbol) [0x00007FF727C2B1D1]
	(No symbol) [0x00007FF727C0B823]
	(No symbol) [0x00007FF727BD75E8]
	(No symbol) [0x00007FF727BD8751]
	GetHandleVerifier [0x00007FF7280847BD+3129501]
	GetHandleVerifier [0x00007FF7280D4D00+3458528]
	GetHandleVerifier [0x00007FF7280CB05D+3418429]
	GetHandleVerifier [0x00007FF727E5687B+844123]
	(No symbol) [0x00007FF727D00AFF]
	(No symbol) [0x00007FF727CFC6D4]
	(No symbol) [0x00007FF727CFC86D]
	(No symbol) [0x00007FF727CEBD79]
	BaseThreadInitThunk [0x00007FF92EF0257D+29]
	RtlUserThreadStart [0x00007FF93006AF08+40]

]]></error>
		<system-out><![CDATA[Tiempo de espera agotado. El elemento no está presente o no es clickeable.
¡Elemento encontrado y clickeado 3 veces con éxito!
¡Elemento encontrado y clickeado 8 veces con éxito!
¡Elemento encontrado y clickeado 3 veces con éxito!
]]></system-out>
	</testcase>
</testsuite>
